index.md

```diff
@@ -94,11 +94,11 @@ description: Vanilla OS is an Immutable Linux-based distribution which aims to p
 <div class="hero hero-feature hero-feature-inverted">
     <div class="container hero-wrapper">
         <div class="hero-text">
-            <h2>Immutable..</h2>
-            <p>..but also not.</p>
-            <p>Vanilla OS is an immutable operating system, core parts of the system are locked down to prevent unwanted changes and corruption from third-party applications or a faulty update. Some paths are still writable, such as the home and configurations directories, this allows the user to keep their files and ensure the normal functioning of applications.</p>
-            <p>Core components are only updated via controlled and atomic transactions, which are applied only on success and made available on reboot.</p>
-            <p>Need to install a component in the core system? No problem, you can open a transactional shell at any time, make your changes and let the system apply them automatically!</p>
+            <h2>Immutable...</h2>
+            <p>...but also not.</p>
+            <p>Vanilla OS is an immutable operating system. Core parts of the system are locked down to prevent unwanted changes or corruption from third-party applications or a faulty update. Some paths are still writable, such as the home and configurations directories, allowing the user to keep their files and  application data.</p>
```

Comment:
```suggestion
            <p>Vanilla OS is an immutable operating system. Core parts of the system are locked down to prevent unwanted changes, corruption from third-party applications or a faulty update. Some paths are still writable, such as the home and configuration directories, allowing the user to keep and alter their files and application data.</p>
```

Reply:
The use of that comma is definitely wrong

Reply:
How? It's a list of things immutability prevents, I don't see an issue with that comma